manifests/pipecd/values.yaml

```diff
@@ -382,15 +382,17 @@ grafana:
           folder: 'Cluster'
           type: file
           disableDeletion: false
-          editable: false
+          # TODO: Disallow to update Grafana UI from the web once got fixed
+          # The following Control-Plane's one should be reverted as well.
+          allowUiUpdates: true
```

Comment:
We can build dashboards by copying the original. But it's a bit tedious especially because of the current number of dashboards.